Apps/Inc/Temperature.h

```diff
@@ -130,5 +138,5 @@ int32_t Temperature_GetMaxTemperature(void);
  * @param DesiredTemp - desired temperature
  * @return FanSpeed: 0-4000 PWM
  */
-int32_t Temperature_PID_Output(int32_t InputTemp, int32_t DesiredTemp);
+uint8_t Temperature_PID_Output(int32_t InputTemp, int32_t DesiredTemp);
```

Comment:
the inputs to this should also be all positive. Shouldn't they be uint32_t?

Reply:
fixed